client/src/app/interfaces/userSearch.ts

```diff
@@ -0,0 +1,8 @@
+export interface UserSearch {
```

Comment:
I think we could use our existing User interface? 
<img width="172" alt="user interface" src="https://user-images.githubusercontent.com/33170291/110226596-a2be0d80-7ea5-11eb-8f65-24c0fd3318e0.PNG">

Reply:
I could but it is better for me to have another interface since in this interface the id is the username (I need the same format as the search stock results